### PR TITLE
Add dependency: smbus2

### PR DESCRIPTION
**Pull request by NicolaiSoeborg**
Fixes:

```python
>>> import bme680                                                                                                                                                                                          
>>> sensor = bme680.BME680(bme680.I2C_ADDR_PRIMARY)                                                                                                                                                        
Traceback (most recent call last):                                                                                                                                                                         
  File "<stdin>", line 1, in <module>                                                                                                                                                                      
  File "/usr/local/lib/python3.7/dist-packages/bme680/__init__.py", line 22, in __init__                                                                                                                   
    import smbus                                                                                                                                                                                           
ModuleNotFoundError: No module named 'smbus'                                                                                                                                                               
```

### CODE REVIEW

**Comment by Gadgetoid** [comment]
Thank you!

I tend to rely on "smbus" from apt since - despite *maintaining* the PyPI package I'm not sure how up-to-date it is with the officially distributed apt package. This is a pain because a PyPI package cannot specify an apt package as a dependency (without some horrible gymnastics that have no place in setup.py).

`smbus2` doesn't provide the `smbus` package, but it might be appropriate to depend on `smbus` here since it seems to work fine and, god knows, I'm never going to be on the ball with fixing/updating it if nobody is using it.

So, TLDR: this should be "smbus" not "smbus2".

**Comment by Gadgetoid** [comment]
Thank you! Now we pray my smbus package isn't subtly broken in some way!
